Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs

```diff
@@ -807,10 +807,10 @@ public static (string c1Control, string code, string [] values, string terminati
 
         if ((!isButtonClicked
              && !isButtonDoubleClicked
-             && ((buttonState == MouseFlags.Button1Pressed)
-                 || (buttonState == MouseFlags.Button2Pressed)
-                 || (buttonState == MouseFlags.Button3Pressed)
-                 || (buttonState == MouseFlags.Button4Pressed))
+             && (buttonState == MouseFlags.Button1Pressed
```

Comment:
Pretty sure this isn't something ReSharper will automatically do, but these can be greatly simplified and turned into a single comparison.